examples/MultiTouchBlink_IRQ/MultiTouchBlink_IRQ.ino

```diff
@@ -0,0 +1,61 @@
+/* 
+ * MultiTouchBlink_IRQ.ino
+ * 
+ * This example shows how to visualise the number of touch points, using the RGB LED. 
+ * 
+ * Whenever a touch event is detected, gigaTouchHandler() records the number of touches and blinks the RGB LED accordingly.
+ * The event is also detected and sent via Serial. The touch handler is detached, blink is performed and then re-attached.
+ *  
+ * For the polling version of this example, see MultiTouchBlink_IRQ.ino
+ * 
+ * Instructions:
+ * 1. Connect your GIGA Display Shield (ASX00039) to a GIGA R1 WiFi (ABX00063) board.
+ * 2. Upload this sketch to your board.
+ * 3. Open the Serial Monitor.
+ * 4. Touch the screen with your finger(s).
+ * 5. The LED will blink, based on how many touch contact are recorded. Note that fingers need to be placed all at the same time. 
+ * 
+ * Initial author: Ali Jahangiri @aliphys
+ * Created: 12 June 2024
+ */
+
+#include "Arduino_GigaDisplayTouch.h"
+
+Arduino_GigaDisplayTouch touchDetector;
+
+int contactPoints = 0;
+
+void gigaTouchHandler(uint8_t contacts, GDTpoint_t* points) {
+  contactPoints = contacts;
+  touchDetector.end();  // close the touch controller
```

Comment:
```suggestion
  touchDetector.detach(); 
```

Reply:
I did not see a member function for `.detach()` in the [Arduino_GigaDisplayTouch.h](https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/blob/main/src/Arduino_GigaDisplayTouch.h) header. When making the change recommended the following error is given.

```
C:\GitHub\Arduino\Arduino_GigaDisplayTouch\examples\MultiTouchBlink_IRQ\MultiTouchBlink_IRQ.ino:30:17: error: 'class Arduino_GigaDisplayTouch' has no member named 'detach'
   touchDetector.detach();  // close the touch controller
                 ^~~~~~

Using library Arduino_GigaDisplayTouch at version 1.0.1 in folder: C:\Users\Ali Jahangiri\Documents\Arduino\libraries\Arduino_GigaDisplayTouch 
Using library Wire in folder: C:\Users\Ali Jahangiri\AppData\Local\Arduino15\packages\arduino\hardware\mbed_giga\4.1.3\libraries\Wire (legacy)
exit status 1

Compilation error: 'class Arduino_GigaDisplayTouch' has no member named 'detach'
```

I agree, having a `detach()` method would be greatly appreciated 😄 ! See https://github.com/arduino-libraries/Arduino_GigaDisplayTouch/issues/8